cluster/saltbase/salt/docker/docker-defaults

```diff
@@ -1,7 +1,8 @@
 {% set grains_opts = grains.get('docker_opts', '') -%}
-{% set e2e_opts = '' -%}
-{% if pillar.get('e2e_storage_test_environment', '').lower() == 'true' -%}
-{% set e2e_opts = '-s devicemapper' -%}
+{% set e2e_opts = '' %}
+{% set e2e_env = (pillar.get('e2e_storage_test_environment', ''))|string() %}
+{% if e2e_env.lower() == 'true' -%}
```

Comment:
This would have made sure that boolean and string both would safely evaluate to the == 'true'. 

Reverting.